build/circle-deps.sh

```diff
@@ -9,6 +9,26 @@ function is_shard() {
   test $(($1 % $CIRCLE_NODE_TOTAL)) -eq $CIRCLE_NODE_INDEX
 }
 
+function fetch_docker() {
+  local image=${1}
+  local tag=${2}
+  local name=${image}:${tag}
+  if ! docker images | grep -q "${name}"; then
+    # If there's a base image cached, load it. A click on CircleCI's "Clear
+    # Cache" will make sure we start with a clean slate.
+    builder="${builder_dir}/builder.${name}.tar"
```

Comment:
`builder` is no longer the right name for this variable. I would say `image` if you weren't using that name above.